cli/tests/pcluster/cli/test_build_image.py

```diff
@@ -112,6 +112,26 @@ def test_error(self, mocker, test_datadir):
             run(command)
         assert_that(exc_info.value.data).is_equal_to(api_response[0])
 
+    def test_no_nodejs_error(self, mocker, test_datadir):
+        """Test expected message is printed out if nodejs is not installed."""
+        mocker.patch(
```

Comment:
Looks like this fits on a single line (?)

Reply:
Done